src/bundle/Resources/views/content/tabs/content.html.twig

```diff
@@ -0,0 +1,22 @@
+{% set languageCode = content.versionInfo.contentInfo.mainLanguageCode %}
+
+{% for contentTypeGroup in contentType.contentTypeGroups %}
```

Comment:
🤔 if you look at `UiContentType:: getFieldDefinitionsByFieldGroup` I am using the `$fieldDefinition->fieldGroup` I thought the translated string could be obtained from the content type but I shall take a look at the mentioned service 👍